### ISSUE TITLE
Error report "attempt to index local 'tooltipData'"

### ISSUE THREAD

**Issue by Taizunz** [Oppzippy/GlobalSearch#20]
Full error report I've been getting for a long while:

```
2x GlobalSearch/Internal/Task.lua:42: ...faceGlobalSearch/Internal/SearchProvider.lua:72: ...rfaceGlobalSearch/Modules/TooltipStorage.lua:55: attempt to index local 'tooltipData' (a nil value)
[string "=[C]"]: in function `error'
[string "@GlobalSearch/Internal/Task.lua"]:42: in function `Poll'
[string "@GlobalSearch/Modules/TaskQueue.lua"]:137: in function `Run'
[string "@GlobalSearch/Modules/TaskQueue.lua"]:109: in function <GlobalSearch/Modules/TaskQueue.lua:108>

Locals:
(*temporary) = "...faceGlobalSearch/Internal/SearchProvider.lua:72: ...rfaceGlobalSearch/Modules/TooltipStorage.lua:55: attempt to index local 'tooltipData' (a nil value)"
```

**Comment by Oppzippy** [comment]
You won't get the error anymore in v0.21.2, but it is not yet fixed, just silently ignored. This means searching for tooltip text may not work, but the addon will be fully functional otherwise.

Please turn on debug mode in Interface > Addons > GlobalSearch > Debug mode and then `/reload`. You should get at least one message in chat saying "C_TooltipInfo.something returned nil with args: some number". Please post a screenshot of those messages in chat (or copy and paste them if you use a chat addon that allows that). If the message is repeated more times than fits in chat without scrolling, just what is visible without scrolling is fine. You can turn debug mode off again after that.

Basically what I need to know is: does the error occur many times or just a few, and if a few, under what specific conditions is it caused.
